### PR TITLE
All PRs for testing

### PR DESCRIPTION
**Pull request by ravanelli**

### CODE REVIEW

**Comment by ravanelli** [comment]
See: https://github.com/coreos/coreos-assembler/pull/3847#issuecomment-2286695755 to in how to test it. You also need coreos-installer to  have https://github.com/coreos/coreos-installer/pull/1511 in the osbuild tree (https://hackmd.io/EjmWWah5T8aT_s1qqNK2lA#To-build-with-custom-coreos-installer)

File: stages/org.osbuild.coreos.osmet [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    :type: str
+    """
+    r = dict()
+    with open(os.path.join(deployed_tree, 'etc/os-release')) as f:
+        for line in f:
+            k, v = line.split('=', 1)
+            r[k] = v.strip(" \n\t'\"")
+    return r
+
+def bwrap(usr, etc, cmd):
```
Comment (jlebon) [inline stages/org.osbuild.coreos.osmet:45]: Can we move this to a utility/separate file and then break down this stage into an osmet stage and a miniso stage that both reuse the bwrap function?

File: stages/org.osbuild.coreos.live-iso
Comment (jlebon) [inline stages/org.osbuild.coreos.live-iso:1]: Let's add `.mono` to this file.

File: platform.live-iso.ipp.yaml [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+            mpp-format-string: '{live_efiboot_img_size_mb * 1024 * 1024}'
+      - type: org.osbuild.coreos.osmet
+        inputs:
+          tree:
+            type: org.osbuild.tree
+            origin: org.osbuild.pipeline
+            references:
+              # XXX: we should rename this pipeline now that we don't use it as
+              # a buildroot
+              - name:build
```
Comment (dustymabe) [inline platform.live-iso.ipp.yaml:70]: I'm pretty sure you just want `- name:tree` here. This is artificially causing the "build" pipeline to get built, which right now we only want/need when we are building RHCOS.

Comment (dustymabe) [inline platform.live-iso.ipp.yaml:70]: I'm pretty sure you just want `- name:tree` here. This is artificially causing the "build" pipeline to get built, which right now we only want/need when we are building RHCOS.

File: platform.live-iso.ipp.yaml [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+              - name: boot
+                type: org.osbuild.ext4
+                source: metal
+                partition:
+                  mpp-format-int: '{image.layout[''boot''].partnum}'
+                target: /tmp-squashfs-dir/boot
+        options:
+          efiboot_img_filename:
+            mpp-format-string: 'efiboot.img'
+      - type: org.osbuild.coreos.osmet
```
Comment (dustymabe) [inline platform.live-iso.ipp.yaml:204]: definitely a bit weird to have this stage called twice? why do we need a minimal-iso?

File: stages/org.osbuild.coreos.osmet (hunk 1 above)
Comment (dustymabe) [reply stages/org.osbuild.coreos.osmet:45]: see also https://github.com/ravanelli/osbuild/pull/1#discussion_r1781628846

**Review by dustymabe — COMMENTED** [review]
some initial thoughts. still going through all of the individual PRs

**Pull request closed without merging**